.devcontainer/Dockerfile

```
@@ -3,21 +3,53 @@
 
 FROM docker.io/ubuntu:24.04
 
+ARG TARGETARCH
+
 ENV NODE_VERSION 22.6.0
 ENV PNPM_VERSION 9.7.0
-ENV NODE_CHECKSUM sha256:f2f4ccbcbc0a443e5fadebd1149a22f96087ec09cef52ff343a15ee835206d96
-ENV PNPM_CHECKSUM sha256:f81316609f23df6cef9b3ed7492a3e0c484fec3a66dfd324884e0ad67ed07581
+
+ENV NODE_CHECKSUM_arm64 c6c7ee62de3637401c15df9a022afaa248d236d59ceca8c188944cf97d0be372
+ENV PNPM_CHECKSUM_arm64 4d65bb31c25fae8813ed5a64abc42f75fb900e678cdb31c4001dacd1a4853c6a
+ENV NODE_CHECKSUM_x64 f2f4ccbcbc0a443e5fadebd1149a22f96087ec09cef52ff343a15ee835206d96
+ENV PNPM_CHECKSUM_x64 f81316609f23df6cef9b3ed7492a3e0c484fec3a66dfd324884e0ad67ed07581
 
 RUN apt-get update \
   && apt-get install -y eza zsh git tig ripgrep bat curl tar micro psmisc \
   && apt-get clean \
   && rm -rf /var/lib/apt/lists/*
-ADD --checksum=$NODE_CHECKSUM https://nodejs.org/dist/v${NODE_VERSION}/node-v${NODE_VERSION}-linux-x64.tar.gz /node.tar.gz
+
+RUN <<EOF
+
+  if [ "$TARGETARCH" = "amd64" ]; then
```

Typo. Should it be `x64`?

```suggestion
  if [ "$TARGETARCH" = "x64" ]; then
```